From:  Panu Matilainen <pmatilai@redhat.com>
Subject: reduce the double separator spec parse error into a warning (#1065563)

See https://bugzilla.redhat.com/show_bug.cgi?id=1065563
--- a/build/parseReqs.c
+++ b/build/parseReqs.c
@@ -166,8 +166,11 @@
 	    if (rpmCharCheck(spec, EVR, ve-v, ".-_+:%{}~")) goto exit;
 
             /* While ':' and '-' are valid, only one of each is valid. */
-	    if (checkSep(EVR, '-', &emsg) || checkSep(EVR, ':', &emsg))
-		goto exit;
+	    if (checkSep(EVR, '-', &emsg) || checkSep(EVR, ':', &emsg)) {
+		rpmlog(RPMLOG_WARNING, _("line %d: %s: %s\n"),
+		       spec->lineNum, emsg, spec->line);
+		emsg = _free(emsg);
+	    }
 
 	    re = ve;	/* ==> next token after EVR string starts here */
 	} else
